README.md

```diff
@@ -9,6 +9,10 @@
 
 https://backpack.github.io/
 
+## Changelog
+
+The up-to-date changelog can be found [here](./CHANGELOG.md)
```

Comment:
How  about
[View our up-to date changelog](...)

Also full-stops please, darling

Reply:
Sounds good, but make sure to preserve `up-to-date`, as @matteo-hertel had already spelled it correctly 🤓

Reply:
![https://media.giphy.com/media/O4EChIxazzrHi/giphy.gif](https://media.giphy.com/media/O4EChIxazzrHi/giphy.gif)

Reply:
@matteo-hertel the gif didn't work 😢